sources/engine/Xenko.VirtualReality/DummyDevice.cs

```diff
@@ -43,6 +43,8 @@ public class DummyDevice : VRDevice
 
         public override TouchController RightHand => null;
 
+        public override TrackedDevice[] TrackedDevices => null;
```

Comment:
I personally prefer collection properties to return empty rather than null.

Reply:
done. now returns new ..[0]